src/com/Jacksonnn/DCCore/Rankup/GuestQuizListener.java

```diff
@@ -41,9 +41,8 @@ public void onInventoryPress(InventoryClickEvent e) {
 //                } else {
                     succeedCheck(player);
 //                }
-            } else {
-                failCheck(player);
             }
+            player.closeInventory();
```

Comment:
This—along with the removal of the `.closeInventory()` calls in `succeedCheck()` and `failCheck()`—prevents the (finite) infinite loop of spamming chat with "you failed the guest quiz!"